docs/en/advanced/Command-Line-Interface.md

```diff
@@ -1,3 +1,24 @@
+New Contents:
+This doc so far only focus on the `alluxio fs` related operations 
+and admin CLI doc only focus on `alluxio fsadmin` related operations.
+
+Other `alluxio [COMMAND other than fs/fsadmin]` is missing in our documentation.
+Some of them are really useful to our users and should be included, such as the 
+usage of `alluxio getConf [key]`. If users know `alluxio getConf --master` can get
+the current alluxio cluster configuration, that would be useful.
+
+The commands include extensions, format, formatMaster, formatWorker, bootstrapConf, 
+fs, fsadmin, getConf, loadufs, logLevel, runClass, runTest, runKVTest, runTests, 
+runJournalCrashTest, runMesosTest, readJournal, upgradeJournal, killAll <WORD>, 
+copyDir, clearCache, confDocGen, thriftGen, protoGen, version, validateConf, validateEnv
+
+Some of them are developer side: confDocGen, thriftGen, protoGen
+We could exclude these commands in this docs, but I will feel better to include all the commands
+and detail their usages as other fs or fsadmin commands docs.
+
+One issue is that we have illustrate those commands in many places in our docs, like runTests, validateEnv.
+To make a single source of truth, we could details commands in this docs and use links in other docs to redirect.
+
 ---
 layout: global
```

Comment:
could we remove the list of operations? The table of contents already lists all the operations, and users can click them to get full descriptions

Reply:
Agree! I never look at the list of operations to get descriptions of commands.

Reply:
I also agree. We should then rename the "real" section from "Example Use Cases" to something like "Operations", since these are not a list of use cases, but the real list of operations with their examples.

Reply:
added two TODOs in CLI.md and Admin-CLI.md
```
// TODO(lu) remove the list of operations
// TODO(lu) change name to Operations
```